CHANGELOG.md

```diff
@@ -22,13 +22,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [`0.7.0-preview`] - 2019-10-11
 
 ### New Known Issues:
-- MSVC v14.23 removes `typeinfo.h` and replaces it with `typeinfo`. This change causes errors when building the Unreal Engine. This issue affects Visual Studio 2019 users. Until [this proposed fix](https://github.com/EpicGames/UnrealEngine/pull/6226) is accepted by Epic Games, you can work around the issue by:
+- MSVC v14.23 removes `typeinfo.h` and replaces it with `typeinfo`. This change causes errors when building the Unreal Engine. This issue **only affects Visual Studio 2019** users. You can work around the issue by:
```

Comment:
Have we made sure that 2017 doesn't come with 14.23 by default too? Just to make sure, I assume not but let's be sure.

Reply:
It didn't come with it last I checked (Friday last week).